jitsi_uploader.sh

```diff
@@ -122,7 +119,15 @@ function process_upload_dir {
 echo $(date) "START Uploader tool received path \"$UPLOAD_DIR\""
 echo $(date) $(ls -l $UPLOAD_DIR 2>&1)
 
-process_upload_dir $UPLOAD_DIR
+# main
+if [[ $UPLOAD_TYPE == "dropbox" ]] && [[ ! -z $TOKEN ]]; then
```

Comment:
I suggest we use a different approach here, and stop checking for token globally.  We should instead check in the dropbox handler whether a token is provided, and exit non-zero there, rather than checking here anymore.